ui.py

```diff
@@ -558,57 +558,100 @@ def open_reports_window(self):
 
-        # Employee selection --------------------------------------
-        employee_label: ttk.Label = ttk.Label(
+        # Print big report button
+        print_class_button: tk.Button = tk.Button(
             report_window,
-            text="Select Employee",
-            font=("Roboto", 12))
+            text="Print All Users Report",
+            command=lambda: self.print_class_report(class_file_name.get(), class_company_name.get(), class_title_name.get(), "11-11-24", "11-16-24"),
+            bg="#00796B",
+            fg="white", font=("Roboto", 12, "bold"),
+            relief="flat", bd=0)
+        print_class_button.pack(pady=20)
+        # --------------------------------------------------------------
 
+
+        # Create the pop-up window for small reports------------------------------------
+        report_window: tk.Toplevel = tk.Toplevel(self.admin_frame)
+        report_window.title("User Reports")
+        report_window.geometry("500x400")
+
+        # file name text input
+        file_name_label: ttk.Label = ttk.Label(report_window, text="File Name",font=("Roboto", 12))
+        file_name_label.pack(pady=5)
+        file_name: tk.Entry = tk.Entry(report_window, width=20)
+        file_name.pack(pady=5)
+
+        # company name text input
+        company_name_label: ttk.Label = ttk.Label(report_window, text="Company Name",font=("Roboto", 12))
+        company_name_label.pack(pady=5)
+        company_name: tk.Entry = tk.Entry(report_window, width=20)
+        company_name.pack(pady=5)
+
+        # title text input
+        title_label: ttk.Label = ttk.Label(report_window, text="Title Name",font=("Roboto", 12))
+        title_label.pack(pady=5)
+        title_name: tk.Entry = tk.Entry(report_window, width=20)
+        title_name.pack(pady=5)
+
+        # student dropdown
+        employee_label: ttk.Label = ttk.Label(report_window, text="Select Employee", font=("Roboto", 12))
         employee_label.pack(pady=10)
 
+        all_users = self.current_user.access.admin_read_all_users()
+        values = {}
+        for user in all_users:
+            values[user[1]] = user[0]
+
         employee_combobox: ttk.Combobox = ttk.Combobox(
             report_window,
-            values=["All Employees", "johndoe", "Luis"])
-
+            values=list(values))
+        
         employee_combobox.pack(pady=5)
-        # -----------------------------------------------------------
 
-        # Print button
-        print_button: tk.Button = tk.Button(
+        # print small report button
+        print_specific_user_button: tk.Button = tk.Button(
             report_window,
-            text="Print Report",
-            command=self.print_report,
+            text="Print Specific User Report",
+            command=lambda: self.print_specific_user_report(file_name.get(), company_name.get(), title_name.get(), "11-11-24", "11-16-24", employee_combobox.get(), str(values[employee_combobox.get()])),
```

Comment:
Again. Make the start date and end date automated

Reply:
I added a calendar for selection of dates (start date and end date)